source/state_representation/include/state_representation/State.hpp

```diff
@@ -89,6 +89,7 @@ class State : public std::enable_shared_from_this<State> {
    */
   void reset_timestamp();
 
+  // FIXME (#50): What is the benefit of declaring those here. We can't declare the data() neither.
```

Comment:
Can you elaborate on this FIXME comment? How is it related to #50?

Reply:
Oups, I didn't finish my own review...See below.

I thought when we put FIXMEs, we put the number of the issue when it was added? Or do we have to open an issue for each FIXME we add?

Reply:
Oops, well that was a miscommunication 😅 FIXME and TODOs don't _need_ an issue link, but if they have one it should point to the relevant issue that describes and discusses the problem. In other words, either the issue exists already and we add a FIXME to remember it in the code, or we find a new FIXME in the code and create a new issue to remember it in our project framework. But adding the "issue related at time of creation" doesn't make much sense. We can always use tools like git blame to find out when a FIXME comment was authored.

It was a recently and informally introduced suggestion so I don't blame you. Also, do you prefer the first or the second format?
```c++
// FIXME(#00): foo
// FIXME (#00): foo
```

IMO the first one is a bit more compact and preferable

Reply:
Ah okay clear now :+1: Yeah the first one is fine for me too